agimus_demos_common/agimus_demos_common/wait_for_non_zero_joints_node.py

```diff
@@ -17,7 +17,7 @@ class WaitForNonZeroJoints(Node):
     def __init__(self):
         super().__init__("wait_for_non_zero_joints_node")
 
-        self.declare_parameter("timeout", 10.0)
+        self.declare_parameter("timeout", 60.0)
```

Comment:
That feels like a long timeout. What takes more than 10 seconds to initialize?

Reply:
Done